Formula/conductr-cli.rb

```diff
@@ -0,0 +1,14 @@
+class ConductrCli < Formula
+  desc "The Lightbend ConductR CLI"
+  homepage "https://conductr.lightbend.com"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.0.1-Mac_OS_X-x86_64.zip"
```

Comment:
Thanks for the pointers. I'm not sure why brew cask is required vs regular brew. Is the intent to signal to the user that they are indeed installing a binary? If so then I'm unsure they'd care that much - most users just copy/paste commands. Feel free to reply or simply close. I appreciate the dialogue.